Dockerfile

```diff
@@ -9,7 +9,7 @@ WORKDIR /jenkins_node_scanner
 COPY Pipfile /jenkins_node_scanner
 COPY Pipfile.lock /jenkins_node_scanner
 RUN pip install --no-cache-dir pipenv
-RUN pipenv install --system
+RUN pipenv install --system --deploy
```

Comment:
I don't understand this change. As mentioned in the previous commit, "pipenv install is an alias for lock + sync", so here we are doing lock (which might update the lockfile) and then `--deploy` will complain that the lockfile has changed? This means that we have to check that the lockfile is up-to-date before building this `Dockerfile`, i.e. we can never build this `Dockerfile` with the exact versions of packages that we want, only the latest... That doesn't sound right.

Reply:
Yes, that is correct. However, I don't know of any other way to do this, because as mentioned, `pipenv sync` does not support the `--system` flag.

Reply:
Also see: https://github.com/pypa/pipenv/issues/2227